portal/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "truenth",
+  "version": "0.0.1",
+  "description": "fills the front-end needs including incorporating i18next for frontend translation needs",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Amy",
+  "license": "ISC",
```

Comment:
I think we're using [BSD as our license](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/LICENSE)

Reply:
k will change